sdks/python/apache_beam/runners/sdf_utils.py

```diff
@@ -38,6 +38,8 @@
 
 if TYPE_CHECKING:
   from apache_beam.io.iobase import RestrictionTracker
+  from apache_beam.io.iobase import RestrictionProgress
```

Comment:
Our linters do not enforce alphanumeric order for modules inside the `TYPE_CHECKING` block, but we should still be diligent about it.  Can you move this up one, please.

Reply:
Done. Thanks for mentioning that!